### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by danbryce**
Added instructions and needed some cleanup for console errors/warnings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,32 +66,28 @@ Quit the server with CONTROL-C.
...
+# while in backend/ from above
+cd ../frontend/web
 npm install
 ```
 
-Start the frontend
+3. Start the frontend
 ```bash
-cd /path/to/pamled/frontend/web
+# while in backend/
```
Comment (jgladwig) [inline README.md:81]: Looks like this should be referencing the frontend web directory:
```suggestion
# while in frontend/web/
```

File: Pipfile [hunk 2]
```diff
@@ -9,11 +9,10 @@ pylint = "*"
 [packages]
 Django = "==3.2.9"
 django-cors-headers = "*"
-django-oso = "*"
```
Comment (jgladwig) [inline Pipfile:12]: What was the reason for removing this? I believe it is required to run the backend.

**Comment by danbryce** [comment]
There were duplicate entries.

**Comment by jgladwig** [comment]
I ran this from a fresh clone. Outside of adding in `django-oso` again I also needed to restrict the `black` package to a version earlier than 22. There appears to be a conflict between `click` versions on a `transcriptic` library and `black`. Limiting black to a previous version seems to have resolved the issues on my end.

So adding to the Pipfile:
```
# Keep black at version under 22 because:
# There are incompatible versions in the resolved dependencies:
# - click<8,>=7.0 (from transcriptic==9.5.0)
# - click>=8.0.0 (from black==22.1.0)
black = "<22"
```

I suspect this is simply due to a change to an unconstrained package requirement in some dependency. `black` appears to have updated to version 22 only a week ago.

This may suggest keeping `Pipfile.lock` that this branch removes even if there are some occasional oddities with regards to new Apple silicon and package versions.

**Pull request merged**